src/DDTrace/Encoder.php

```diff
@@ -2,10 +2,12 @@
 
 namespace DDTrace;
 
+use DDTrace\Contracts\Span as SpanInterface;
```

Comment:
Not required any change, but I noticed that we do the `use DDTrace\Contracts\Span as SpanInterface;` quite often.
Given that we are probably going to use the interfaces more than concrete (hopefully 😄) what do you think about adopting the following convention for the future?

```
// If only contract used
use DDTrace\Contracts\Span;

// If only concrete used
use DDTrace\Span;

// If both used
use DDTrace\Contracts\Span;
use DDTrace\Span as DDSpan;
```
?